schemas/vscs.json

```diff
@@ -514,8 +514,16 @@
                 "items": {
                   "type": "string"
                 },
-                "sectionEnd": "VSC Options",
                 "propertyOrder": 570
+            },
```

Comment:
sample_deployment.xlsx is an empty file.  Something looks wrong about that.  Could you investigate?

Reply:
I was able to verify that sample_deployment is indeed valid.